src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java

```diff
@@ -0,0 +1,5 @@
+package wolox.training.exceptions;
+
+public class BookAlreadyOwnedException extends RuntimeException {
+
```

Comment:
You could add a descriptive message of error, by adding this sentence:
super(MENSAJE);

This MENSAJE is a String.

Reply:
I added the message